### PR TITLE
Added contributing.md link

### PR DESCRIPTION
**Pull request by gurumukhi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ To publish to __https://your_username.github.io/aframe/__:
 
     npm run ghpages your_username
 
+## Contributing
+
+Get involved! Check out the [Contributing Guide](CONTRIBUTING.md) on how to get started!
```
Comment (ngokevin) [inline README.md]: Oops, maybe change the last exclamation mark to a period. So we don't yell it twice :)

Comment (ngokevin) [inline README.md]: oops, `on how to get started` to `for how to get started`

**Comment by gurumukhi** [comment]
@ngokevin i think we are finally good to merge this ;-)

**Comment by mkungla** [comment]
@gurumukhi can you please `squash` your commits into one

**Comment by ngokevin** [comment]
We can squash from GitHub :) 

Thanks!

**Comment by gurumukhi** [comment]
Thanks. 

@ngokevin, For my information.. can I squash the commits on github UI itself?

**Comment by ngokevin** [comment]
There's an option to squash during the merge process on GitHub. I just did that when merging.

**Comment by mkungla** [comment]
@gurumukhi nothing to worry about. I did not know that github has squash opt for PR's

**Comment by gurumukhi** [comment]
@mkungla I also didn't know :-D

& Thanks @ngokevin for letting us know.

**Pull request merged**